packages/ui/.storybook/StoryHelper.css

```diff
@@ -13,7 +13,5 @@ body,
 }
 
 .test-area {
-  align-items: center;
```

Comment:
Does this mess up the other storybook views?  Probably better to avoid this and add some css specific to the form builder component to make it behave appropriately.

Reply:
Totally does, I was planning to put that back.. now seems like a good time :)